templates/iot/examples/postgresql/iot-config.yaml

```diff
@@ -0,0 +1,18 @@
+kind: IoTConfig
+apiVersion: iot.enmasse.io/v1alpha1
+metadata:
+  name: default
+spec:
+  # ...
```

Comment:
We should make this fully workable example. Probably a default one.

Reply:
I would prefer to provide an "managed" version, by default, in the next step. That was the reason for not putting a complete one here. As it would repeat other settings, which are not relevant to Postgres.